### ISSUE TITLE
fix cross app bts not working

### ISSUE THREAD

**Issue by sys13** [Appdynamics/AppDash#1]
Fails on this query if it doesn't match any applications, but doesn't tell user
```
SELECT bt, art, cpm, epm FROM applications WHERE app REGEXP ".*Gestion.*" AND bt REGEXP "ServiceReception.*"
```

**Comment by sys13** [comment]
fixed by 70b4753fb0bd166ba3aaa1733119edf5d08ea20b

**Comment by sys13** [comment]
Fixed by 70b4753fb0bd166ba3aaa1733119edf5d08ea20b
